domain/src/main/java/org/oppia/android/domain/feedbackreporting/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+"""
+Utilities corresponding to feedback reporting.
```

Comment:
These aren't quite utilities. Suggest instead:

``Domain services & definitions corresponding to user feedback reporting.``

to be a bit more specific.

Reply:
done